### PR TITLE
Updating the readme and changelog for Jetpack 13.3-a.3

### PR DESCRIPTION
**Pull request by zinigor**
Fixes #

## Proposed changes:

*

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Go to '..'
*

### CODE REVIEW

File: projects/plugins/jetpack/CHANGELOG.md [hunk 1]
```diff
@@ -4,20 +4,15 @@
...
 ## 13.3-a.3 - 2024-03-18
 ### Enhancements
-- Add error log on ssh invite [#36416]
-- Adding Native (Web Share) button to Sharing Buttons block [#35967]
-- AI Assistant: provide per block quick actions [#36393]
-- Blog Stats and Top Posts & Pages block: ensure blocks do not render in email newsletters. [#36372]
-- Only show installation errors on plugins page [#36390]
-- Share buttons: Add a Bluesky sharing button [#36181]
-- Update sso copy when inviting users [#36385]
+- AI Assistant: provide per block quick actions to make them more relevant. [#36393]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- AI Assistant: Provide per-block quick actions to make them more relevant. [#36393]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 2]
```diff
@@ -4,20 +4,15 @@
...
+- General: Only show installation errors on plugins page. [#36390]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- General: Only show installation errors on Plugins page. [#36390]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 3]
```diff
@@ -4,20 +4,15 @@
...
+- Newsletters: ensure blog stats and top posts blocks do not render in email newsletters. [#36372]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Newsletters: Ensure blog stats and top posts blocks do not render in email newsletters. [#36372]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 4]
```diff
@@ -28,6 +23,9 @@
 - Fix minor UI issues in premium-content block editor [#36398]
 - Jetpack_Portfolio: Prevent a division by 0 fatal when a user uses the shortcode with columns=0 [#36433]
 - Publicize: prevent panel from jumping after activation [#36362]
+- Sharing Block: remove extra margin previously added to the first button. [#36386]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Sharing Block: Remove extra margin previously added to the first button. [#36386]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 5]
```diff
@@ -37,53 +35,50 @@
...
 
 ### Other changes <!-- Non-user-facing changes go here. This section will not be copied to readme.txt. -->
-- Add site migration endpoint [#36253]
-- Compile typescript extensions in Jetpack [#35904]
-- Contact Form: add deprecation warnings to Contact Form module codebase [#36040]
-- Contact Form: refactor editor to use forms package [#36089]
-- Contact Form: refactor endpoints to use forms package [#36087]
+- Add metadata to the post to better diagnose need for Reader and Firehose. [#36254]
+- Add site migration endpoint. [#36253]
+- Compile typescript extensions in Jetpack. [#35904]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]: Is this something we usually include in changelogs? If so:

```suggestion
- Compile TypeScript extensions in Jetpack. [#35904]
```

Comment (zinigor) [reply projects/plugins/jetpack/CHANGELOG.md]: Yes, it's in the Other section which will not get published to the official readme.txt.

File: projects/plugins/jetpack/CHANGELOG.md [hunk 6]
```diff
@@ -37,53 +35,50 @@
...
+- Contact Form: add deprecation warnings to Contact Form module codebase. [#36040]
+- Contact Form: refactor editor to use forms package. [#36089]
+- Contact Form: refactor endpoints to use forms package. [#36087]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Contact Form: Add deprecation warnings to Contact Form module codebase. [#36040]
- Contact Form: Refactor editor to use Forms package. [#36089]
- Contact Form: Refactor endpoints to use Forms package. [#36087]
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 7]
```diff
@@ -37,53 +35,50 @@
...
 - Editor: Update to the most recent version of the @automattic/calypso-color-schemes package. [#36227]
-- General: update to the most recent version of @automattic/calypso-color-schemes [#36187]
+- General: update to the most recent version of @automattic/calypso-color-schemes. [#36187]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- General: Update to the most recent version of @automattic/calypso-color-schemes. [#36187]
```

File: projects/plugins/jetpack/readme.txt [hunk 8]
```diff
@@ -328,20 +328,15 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
...
-- Add error log on ssh invite
-- Adding Native (Web Share) button to Sharing Buttons block
-- AI Assistant: provide per block quick actions
-- Blog Stats and Top Posts & Pages block: ensure blocks do not render in email newsletters.
-- Only show installation errors on plugins page
-- Share buttons: Add a Bluesky sharing button
-- Update sso copy when inviting users
+- AI Assistant: provide per block quick actions to make them more relevant.
+- General: Only show installation errors on plugins page.
+- Newsletters: ensure blog stats and top posts blocks do not render in email newsletters.
```
Comment (tbradsha) [inline projects/plugins/jetpack/readme.txt]:
```suggestion
- AI Assistant: Provide per-block quick actions to make them more relevant.
- General: Only show installation errors on plugins page.
- Newsletters: Ensure blog stats and top posts blocks do not render in email newsletters.
```

File: projects/plugins/jetpack/CHANGELOG.md [hunk 9]
```diff
@@ -37,53 +35,50 @@
...
-- Subscription Site: Polishing the Subscribe block toggle [#36177]
-- Subscription Site: Update Subscribe block after the post nudge [#36107]
-- Update code references in docs and comments [#36234]
-- Updated package dependencies. [#36309]
-- Updated package dependencies. [#36325]
-- Update lockfiles [#36195]
+- Subscription Site: Polishing Newsletter settings. [#36218]
+- Subscription Site: Polishing the Subscribe block toggle. [#36177]
+- Subscription Site: Update Subscribe block after the post nudge. [#36107]
+- Uncheck users-new.php Invite user on load if WooCommerce plugin is activated. [#36140]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]: ?

File: projects/plugins/jetpack/CHANGELOG.md [hunk 10]
```diff
@@ -37,53 +35,50 @@
...
 - Removed the feature flag for newsletter categories settings. [#36073]
 - RNMobile: Remove code associated with Story block for the mobile native version. [#36151]
-- Show Browse sites when wp-admin interface is selected and using nav unification [#36198]
-- Subscriber Login: Prevent the HTTP 301 redirection for Atomic sites [#36311]
-- Subscriber Login: Remove the premium content cookie specified for the blog on logging out [#35441]
-- Subscriptions: fix empty email for pending confirm paywall on .com [#36321]
+- Show Browse sites when wp-admin interface is selected and using nav unification. [#36198]
+- Subscriber Login: Prevent the HTTP 301 redirection for Atomic sites. [#36311]
+- Subscriber Login: Remove the premium content cookie specified for the blog on logging out. [#35441]
+- Subscriptions: fix empty email for pending confirm paywall on .com. [#36321]
```
Comment (tbradsha) [inline projects/plugins/jetpack/CHANGELOG.md]:
```suggestion
- Subscriptions: Fix empty email for pending confirm paywall on .com. [#36321]
```

**Review by tbradsha — DISMISSED** [review]

File: projects/plugins/jetpack/CHANGELOG.md (hunk 9 above)
Comment (zinigor) [reply projects/plugins/jetpack/CHANGELOG.md]: Yeah, I'll have to re-word that and put it in its place alphabetically.

**Review by tbradsha — APPROVED** [review]

**Pull request merged**